syz-manager/manager.go

```diff
@@ -1066,7 +1064,170 @@ func (mgr *Manager) fuzzerConnect() ([]rpctype.RPCInput, BugFrames, bool) {
 	for frame := range mgr.dataRaceFrames {
 		dataRaceFrames = append(dataRaceFrames, frame)
 	}
-	return corpus, BugFrames{memoryLeaks: memoryLeakFrames, dataRaces: dataRaceFrames}, mgr.coverFilterFilename != ""
+	var err error
+	once.Do(func() {
+		err = mgr.updateKernelModules(a)
+		if err != nil {
+			return
```

Comment:
I think we need to log.Fatalf here. This will only return error to the fuzzer, but once is already called, so it won't be called again. As the result we will end up with broken state: modules are not initaliazed, mgr.ready is not set, but we will never try again and continue to work as everything is fine.